src/emmetHelper.ts

```diff
@@ -948,7 +978,7 @@ export function updateExtensionsPath(emmetExtensionsPath: string, workspaceFolde
 		});
 	});
 
-	return Promise.all([snippetsPromise, variablesFromFile]).then(() => Promise.resolve());
+	return Promise.all([snippetsPromise, variablesPromise]).then(() => Promise.resolve());
```

Comment:
This was a typo because `variablesPromise` was unused and `variablesFromFile` is an object why would you resolve it.